hera_mc/geo_handling.py

```diff
@@ -19,298 +21,397 @@
+        found_location = []
+        for L in to_find:
+            station_name = False
+            try:
+                antenna_number = int(L)
+                station_name = self.find_station_of_antenna(antenna_number, query_date)
+            except ValueError:
+                station_name = L.upper()
+            found_it = False
+            if station_name:
+                for a in self.session.query(geo_location.GeoLocation).filter(geo_location.GeoLocation.station_name == station_name):
+                    for key in self.station_types.keys():
+                        if a.station_name in self.station_types[key]['Stations']:
+                            this_station = key
+                            break
+                        else:
+                            this_station = 'No station type data.'
+                    a.gps2Time()
+                    desc = self.station_types[this_station]['Description']
+                    ever_connected = self.is_in_connections(a.station_name, '<')
+                    active = self.is_in_connections(a.station_name, query_date)
+                    found_it = True
+                    hera_proj = Proj(proj='utm', zone=a.tile, ellps=a.datum, south=True)
+                    a.lon, a.lat = hera_proj(a.easting, a.northing, inverse=True)
+                    found_location.append(copy.copy(a))
+                    if show_location:
+                        if verbosity == 'm' or verbosity == 'h':
+                            print('station_name: ', a.station_name)
+                            print('\teasting: ', a.easting)
+                            print('\tnorthing: ', a.northing)
+                            print('\tlon/lat:  ', a.lon, a.lat)
+                            print('\televation: ', a.elevation)
+                            print('\tstation description ({}):  {}'.format(this_station, desc))
+                            print('\tever connected:  ', ever_connected)
+                            print('\tactive:  ', active)
+                            print('\tcreated:  ', cm_utils._get_displayTime(a.created_date))
+                        elif verbosity == 'l':
+                            print(a, this_station)
+            if show_location:
+                if not found_it and verbosity == 'm' or verbosity == 'h':
+                    print(L, ' not found.')
+        return found_location
+
+    def get_all_everconnected_locations(self):
+        stations = self.session.query(geo_location.GeoLocation).all()
+        connections = self.session.query(part_connect.Connections).all()
+        stations_conn= []
         for stn in stations:
             hera_proj = Proj(proj='utm', zone=stn.tile, ellps=stn.datum, south=True)
             stn.lon, stn.lat = hera_proj(stn.easting, stn.northing, inverse=True)
-            ever_connected = geo_location.is_in_connections(args, stn.station_name,'>')
+            ever_connected = self.is_in_connections(stn.station_name,'>')
```

Comment:
Does passing '>' for active_date work? It looks like it gets ignored if it's not an astropy Time object.